test/e2e/git_test.go

```diff
@@ -82,8 +82,8 @@ spec:
 			Template: &v1alpha1.AppStatusTemplate{
 				ExitCode: 0,
 			},
-			ConsecutiveReconcileSuccesses: 1,
-			ObservedGeneration:            1,
+			ConsecutiveReconcileSuccesses: 2,
```

Comment:
Mainly highlighting how behavior has changed with updates. This behavior should not change for this feature.